src/components/TextWrapper.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      margin: '0 0',
+    },
+    items: {
+      display: 'flex',
+      flex: '1',
+      justifyContent: 'space-between',
+      margin: '2px 0px',
+    },
+    titleText: {
+      color: '#6A6A6A',
+      margin: '10px 0',
+    },
+    bold: {
+      fontWeight: 700,
+    },
+  }),
+);
+
+interface TextWrapperProps {
+  title?: string;
+  numbers: number[];
+  labels: string[];
+  units?: string[];
+  color?: 'inherit' | 'primary' | 'textSecondary';
+  bold?: boolean;
+}
+
+export default function TextWrapper(props: TextWrapperProps): JSX.Element {
+  const classes = useStyles();
+
+  const getLabeledNumber = (number: number, label: string, index: number) => {
+    return (
+      <div key={index} className={classes.items}>
+        {props.bold ? (
+          <>
+            <Typography className={classes.bold} color={props.color ? props.color : 'inherit'} variant="h4">
+              {label}
+            </Typography>
+            <Typography className={classes.bold} color={props.color ? props.color : 'inherit'} variant="h4">
+              {number + (props.units ? props.units[index] : '')}
+            </Typography>
+          </>
+        ) : (
+          <>
+            <Typography color={props.color ? props.color : 'inherit'} variant="h4">
+              {label}
+            </Typography>
+            <Typography color={props.color ? props.color : 'inherit'} variant="h4">
+              {number + (props.units ? props.units[index] : '')}
+            </Typography>
+          </>
```

Comment:
I don't think we need to wrap such a large ternary for this. I believe something along the lines of

```
const typographyClasses = props.bold ? classes.bold : null
...
...
<Typography className={typographyClasses}...> ... < />
...
```

can be done

Reply:
ooh yeah definitely i made some changes to clean this up and will push once i address all the comments.